docs/scripting-element-object.rst

```diff
@@ -0,0 +1,566 @@
+.. _splash-element:
+
+Element Object
+==============
+
+Element objects are created by :ref:`splash-select` and :ref:`splash-select-all`.
```

Comment:
How hard is it to fix that? I think it is better to do before the merge because we'll get a backwards compatibility issue if we change it later.